app/src/containers/ListOfNews/index.js

```diff
@@ -44,9 +42,9 @@ class ListOfNews extends React.Component {
       };
     }
 
-    getNewsListAPI(callback) {
-        const url = new URL(window.location.protocol + '//' + window.location.hostname + this.state.portOption + "/api/v1/news");
-
+    //getNewsListAPI(callback) {
```

Comment:
You have committed the change with removing news GET and added the hard-coded file... 
So this will be going to the production code, which should not be happened. 
It's OK to have a mock response (news.json) locally for testing purposes, and your code should not be depending on that.
Maybe you can introduce another execution path for local testing, but this will break the news functionality.....